airbyte-integrations/connectors/source-tempo/acceptance-test-config.yml

```diff
@@ -1,19 +1,40 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-tempo:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_tempo/spec.json"
+    tests:
+      - spec_path: "source_tempo/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.2.6"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        expect_records:
+          path: "integration_tests/expected_records.txt"
```

Comment:
were the expected records produced before or after the migration?

Reply:
Not quite, but there's a reason for that. Originally I generated them before the migration but within this request I have also made a migration to a new API version which made the response a little bit different from the one we used to have. Lots of links changed their values (you could notice schemas changed as well), at least the API version which is present in almost every link (in almost every record there's a link pointing to itself; besides, there's a lot of nested objects containing links as well). A sanity check was made to be sure things work as expected - a number of records yielded by each stream remains the same